tests/core/providers/test_websocket_provider.py

```diff
@@ -270,11 +270,13 @@ async def test_listen_event_awaits_msg_processing_when_subscription_queue_is_ful
     sub_request_information = RequestInformation(
         method=RPCEndpoint("eth_subscribe"),
         params=["mock"],
-        response_formatters=(),
+        response_formatters=[[], [], []],
         subscription_id=sub_id,
     )
     async_w3.provider._request_processor._request_information_cache.cache(
-        "", sub_request_information
+        # cache key is the result of `generate_cache_key` with the sub_id as the arg
+        "0138b5d63d66121d8a6e680d23720fa7",
```

Comment:
may be worth just calculating that here 🤷 
```suggestion
        generate_cache_key(sub_id)
```

Reply:
A little more overhead for a test with an expected result, but makes it clearer 👍🏼